tests/subscriptions/test_worker.py

```diff
@@ -1,124 +1,133 @@
 from concurrent.futures import ThreadPoolExecutor
```

Comment:
It is probably slightly simpler to review this file with whitespace changes disabled: https://github.com/getsentry/snuba/pull/981/files?diff=split&w=1#diff-1f1ab4f01202fc20f7b4838bac6d31de